packages.yml

```diff
@@ -0,0 +1,3 @@
+packages:
```

Comment:
Should we consider just using 0.8.0 straight away? Any reason why not?

Reply:
good point, changed it now